R/dataframe__frame.R

```diff
@@ -1366,3 +1366,15 @@ DataFrame_pivot = function(
     # unwrap and add method context name
     unwrap("in $pivot():")
 }
+
+#' @title Rename columns of a DataFrame
+#' @keywords DataFrame
+#' @param mapping A named list like `list(new = "old")`.
+#' @return DataFrame
+#' @examples
+#' pl$DataFrame(mtcars)$
+#'   lazy()$
+#'   rename(list(miles_per_gallon = "mpg", horsepower = "hp"))
+DataFrame_rename = function(mapping) {
+  self$lazy()$rename(mapping)$collect()
+}
```

Comment:
Would it be possible to omit list() from the argument by changing it like follows?

```suggestion
DataFrame_rename = function(...) {
  self$lazy()$rename(list(...))$collect()
}
```

Reply:
I've got to go now so I'll take a look later

Reply:
done